_rules/invalid-form-field-value-36b590.md

```diff
@@ -0,0 +1,203 @@
+---
+id: 36b590
+name: Error message describes invalid form field value
+rule_type: atomic
+description: |
+  This rule checks that text descriptions are provided when the user completes a form field with information that is not an allowed value or using a not allowed format.
+accessibility_requirements:
+  wcag20:3.3.1: # Error Identification (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G84: # Providing a text description when the user provides information that is not in the list of allowed values
+    forConformance: false
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+  wcag-technique:G85: # Providing a text description when user input falls outside the required format or values
+    forConformance: false
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Language
+acknowledgements:
+  authors:
+    - Carlos Duarte
+    - João Vicente
+---
+
+## Applicability
+
+The rule applies to each HTML [input element][] with the [type attribute][] in one of the following [states][]: "Text", "Telephone", "URL", "E-mail", "Password", "Date", "Month", "Week", "Time", "Local Date and Time", and "Number";
```

Comment:
Also, the SC only applies to input errors that are automatically detected.

Add an inapplicable test case?

Reply:
But that is not explicitly in the applicability of this rule (it is implicitly in the first expectation).